terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,12 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+max_task_count = 1
+# service configs
+log_level = "debug"
+
+# scaling configs
+service_scaledown_schedule = "55 19 * * ? *"
+service_scaleup_schedule   = "5 6 * * ? *"
+
+required_cpus    = 3840
```

Comment:
I would then remove these for CIDEV profile as it would be picked up by the default

Reply:
thanks @Ngriffiths1  I have made the change.